client/src/components/main/chatwindow/EditComment.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useContext } from 'react';
+import styled from 'styled-components';
+import Styled from './styles/comment.styles';
+import axios from 'axios';
+import { ChatContext } from './ChatWindow';
+
+/*
+Posts new comment to database when enter key pressed and triggers re-render by updating ChatWindow state
```

Comment:
Are we posting here?  This is a `.patch()`

Reply:
Yes, you're right, should be patch